tests/Gedmo/Tree/MaterializedPathUuidORMTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Doctrine Behavioral Extensions package.
+ * (c) Gediminas Morkevicius <gediminas.morkevicius@gmail.com> http://www.gediminasm.org
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gedmo\Tests\Tree;
+
+use Doctrine\Common\EventManager;
+use Gedmo\Tests\Tool\BaseTestCaseORM;
+use Gedmo\Tests\Tree\Fixture\MPCategoryUuid;
+use Gedmo\Tree\TreeListener;
+use Symfony\Component\Uid\UuidV4;
+
+/**
+ * These are tests for Tree behavior when using Uuid as primary key and TreePathSource.
+ *
+ * @author Gustavo Falco <comfortablynumb84@gmail.com>
+ * @author Gediminas Morkevicius <gediminas.morkevicius@gmail.com>
+ * @author Andrea Bergamasco <a.bergamasco@keesystem.com>
+ */
+final class MaterializedPathUuidORMTest extends BaseTestCaseORM
+{
+    /**
+     * @var array<string, mixed>
+     */
+    protected $config;
+
+    /**
+     * @var TreeListener
+     */
+    protected $listener;
```

Comment:
```suggestion
    private TreeListener $listener;
```

Reply:
Thanks, fixed in e316a1ed

Reply:
Please, remove the docblock. It does not add any value here.

Reply:
Thanks, fixed in 3352422b440732a01220eb8b4b307df878a7886d